docs/integrations/builtin/credentials/facebookleadads.md

```diff
@@ -31,18 +23,40 @@ View [example workflows and related content](https://n8n.io/integrations/faceboo
 
 ## Using OAuth2
 
-To configure this credential, you'll need:
+To configure this credential, you'll need a [Meta for Developers](https://developers.facebook.com/){:target=_blank .external-link} account and:
 
 - A **Client ID**
 - A **Client Secret**
 
-In your app, be sure you have either the **Facebook Login** or **Facebook Login for Business** product added. Then:
-
-1. Within that product, go to **Settings**.
-2. From the n8n credential, copy the **OAuth Redirect URL**.
-3. Within Facebook, paste that URL into the **Valid OAuth Redirect URIs** field and Save.
-4. Set your Facebook app to **Live** App Mode.
-5. In **App settings > Basic**, copy the **App ID**. Use this as the **Client ID** within the n8n credential.
-6. In **App settings > Basic**, copy the **App Secret**. Use this as the **Client Secret** within the n8n credential.
+To get both, [create a Meta app](https://developers.facebook.com/docs/development/create-an-app){:target=_blank .external-link} with either the Facebook Login product or the Facebook Login for Business product.
+
+To create your app and set up the credential with **Facebook Login for Business**:
+
+1. Go to the Meta Developer [App Dashboard](https://developers.facebook.com/apps){:target=_blank .external-link} and select **Create App**.
+2. If you have a business portfolio and you're ready to connect the app to it, select the business portfolio. If you don't have a business portfolio or you're not ready to connect the app to the portfolio, select **I don’t want to connect a business portfolio yet** and select **Next**. The **Use cases** page opens.
+3. Select **Other**, then select **Next**.
+4. Select **Business** and **Next**.
+5. Complete the essential information:
+    * Add an **App name**.
+    * Add an **App contact email**.
+    * Here again you can connect to a business portfolio or skip it.
+1. Select **Create app**.
+1. The **Add products to your app** page opens.
```

Comment:
I'd maybe collapse this into the step above since it's not a user action (similar to "The Settings page for this product opens." in the next step).